### ISSUE TITLE
ME2LE issue when exporting wwise BNK files 

### ISSUE THREAD

**Issue by Xehanort88** [Mgamerz/ME3Explorer#426]
**Describe the bug**
Extracting bnk files with the Explorer from ME2LE adds to the header some junk bits that hamper the quality of the file itself.

**To Reproduce**
Open package Explorer for LE2, open Bio_S_endgame2.pcc and extract the wwise bank files. They all have these extra bytes before BKHD which is meant to be the bank header

**Expected behavior**
Bnk files to be extracted without these extra bytes?

**Screenshots and files used to replicate this**
Please see file attached and open with HXD or even Notepad++

**Version information:**
The one updated a month ago.

[Default__WwiseBank.zip](https://github.com/user-attachments/files/16639146/Default__WwiseBank.zip)

**Comment by Mgamerz** [comment]
Please provide exact steps. "Extract the bank" is ambiguous.

**Comment by Xehanort88** [comment]
Right-click on bnk file "export bank to file" with Soundplorer; right-click, export, "embedded files" with package editor.
